syntax = "proto2";

package Codec_AddressBook;

message AddressBookEntry
{
	required bytes scrAddr = 1;
	repeated bytes txHash = 2;
}

message AddressBook
{
	repeated AddressBookEntry entry = 1;
}